extra_data/cli/make_virtual_cxi.py

```diff
@@ -7,26 +7,20 @@
 from textwrap import dedent
 
 from extra_data import RunDirectory
-from extra_data.components import XtdfDetectorBase
+from extra_data.components import detectors
 from extra_data.exceptions import SourceNameError
 
 log = logging.getLogger(__name__)
 
 
 def _get_detector(data, min_modules):
-    for cls in XtdfDetectorBase.__subclasses__():
+    for cls in detectors.list:
```

Comment:
I think we could use the `identify_multimod_detectors` function here and get rid of this little function.

Reply:
Yes, thank you, implemented in https://github.com/European-XFEL/EXtra-data/commit/156b8bf353b59b107950a2baf566352c061acc4c.